.github/workflows/deploy-to-prod-from-default.yml

```diff
@@ -52,20 +52,16 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: E2E test on prod
+      - name: Run Cypress E2E Tests
+        # We run the tests through the command, and not using cypress-io/github-action@v6, because this is the only way to hide the logs.
```

Comment:
```suggestion
        # We run the tests through the command, and not using cypress-io/github-action@v6, because this is the only way to hide the logs that might contain secrets.
```